specific/firefox/cliqz@cliqz.com/bootstrap.js

```diff
@@ -8,6 +8,40 @@ function startup(aData, aReason) {
   Services.scriptloader.loadSubScript('chrome://cliqz/content/CLIQZ.js', global);
   CLIQZ = global.CLIQZ;
   CLIQZ.start(aData, aReason);
+
+  Components.utils.import('resource://gre/modules/Timer.jsm');
+  // Give some time to the telemetry controller to initialize at startup
+  setTimeout(reportTelemetry, 5000);
```

Comment:
This is still very early in startup and `getCurrentPingData()` can be a bit expensive.
Can we delay this to:
- 60 seconds after startup
- on "profile-before-change" (if shutting down before hitting the 60 seconds)

That roughly matches our Telemetry initialization.